greedybear/cronjobs/db_backup.py

```diff
@@ -0,0 +1,45 @@
+import os
+from datetime import datetime
+
+from cStringIO import StringIO
+from django.core import management
+
+from greedybear import settings
+from greedybear.apps import GreedyBearConfig
+from greedybear.cronjobs.base import Cronjob
+
+
+class DbBackup(Cronjob):
+    def __init__(self):
+        super().__init__()
+        self.interval = settings.DB_BACKUP_FREQUENCY
+        self.location = (
+            str(settings.DB_BACKUP_STORAGE_OPTIONS["location"]) or "/opt/backups"
+        )
+        self.max_backups = int(settings.DB_BACKUP_STORAGE_OPTIONS["max_backups"]) or 10
+
+    @property
+    def minutes_back_to_lookup(self):
+        return self.interval
+
+    def _remove_oldest_backup(self, dir):
+        # removes files older than self.max_backups
+        files = os.listdir(dir)
+        if len(files) >= self.max_backups:
+            oldest_file = min(files, key=os.path.getctime)
+            os.remove(os.path.abspath(oldest_file))
+
+    def run(self):
+        self._remove_oldest_backup(self.location)
```

Comment:
no need to pass this value as a variable...

Reply:
Oh Right we can access it from the method itself 👍🏼